modules/olm-cluster-capability.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/cluster-capabilities.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="olm-cluster-capability_{context}"]
+= Operator Lifecycle Manager capability
+
+[discrete]
+== Purpose
+
+Operator Lifecycle Manager helps users install, update, and manage the lifecycle of Kubernetes native applications (Operators) and their associated services running across their {product-title} clusters. It is part of the Operator Framework, an open source toolkit designed to manage Operators in an effective, automated, and scalable way.
```

Comment:
```suggestion
Operator Lifecycle Manager helps users install, update, and manage the lifecycle of Kubernetes native applications (Operators) and their associated services running across their {product-title} clusters. It is part of the Operator Framework, an open-source toolkit designed to manage Operators effectively in an automated and scalable way.
```

Reply:
Updated, although perhaps "open source" should remain? 
https://redhat-documentation.github.io/supplementary-style-guide/#open-source

Reply:
@rohennes Thanks for the response. IMO, we use `open-source` in our OCP docs. I do not see occurrences of `open source`.  I was unable to find it when I searched using this keyword.

@bmcelvee Can you PTAL and let us know your thoughts on this? TY!

Reply:
Based on the supplementary style guide, it's "open source": https://redhat-documentation.github.io/supplementary-style-guide/#open-source